### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by natakazakova**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,18 +1,76 @@
...
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+
+# Pivot Grid for WPF - How to Connect a Pivot Grid to an OLAP Data Source
+
+The [Pivot Grid](7228) can display data from the the OLAP server (Microsoft Analysis Services). This example shows how to specify connection settings to the server and create fields that relate to specific measures and dimensions of the cube.
+
+To bind the Pivot Grid control to an OLAP cube, follow the steps below.
+
+1. Specify connection settings to the server. You can do it in the [Items Source Wizard](https://docs.devexpress.com/WPF/8015/controls-and-libraries/pivot-grid/binding-to-data/olap-data-source/binding-to-olap-data-sources#bind-to-an-olap-cube-at-design-time). The following code shows the resulted XML.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
1. Specify connection settings to the server. You can do it in the [Items Source Wizard](https://docs.devexpress.com/WPF/8015/controls-and-libraries/pivot-grid/binding-to-data/olap-data-source/binding-to-olap-data-sources#bind-to-an-olap-cube-at-design-time). The following code shows the resulting XML:

```

File: Readme.md [hunk 2]
```diff
@@ -1,18 +1,76 @@
...
+   <!-- xmlns:dxpg="http://schemas.devexpress.com/winfx/2008/xaml/pivotgrid" -->
+    <Grid>
+        <dxpg:PivotGridControl Name="pivotGridControl1" RowTreeMinWidth="170"
+                               OlapDataProvider="Adomd" 
+                               OlapConnectionString="{Binding ConnectionString, Source={StaticResource PivotOlapDataSource}}" />
+    </Grid>
+   ```
+
+
+1. Create fields for all the measures and dimension in the bound OLAP cube, and moves these fields to the specified area, making them hidden. To do it, use the [PivotGridControl.RetrieveFields](https://docs.devexpress.com/WPF/DevExpress.Xpf.PivotGrid.PivotGridControl.RetrieveFields.overloads) method overload with the _visible_ parameter and set the field's visibility to `false`. The `RetrieveFields` method generates [DataSourceColumnBinding](https://docs.devexpress.com/WPF/DevExpress.Xpf.PivotGrid.DataSourceColumnBinding?v=22.1) objects for each Pivot Grid field in OLAP, Server, and Optimized [modes](https://docs.devexpress.com/CoreLibraries/403802/devexpress-pivot-grid-core-library/pivot-grid-modes?v=22.1).
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
1. Create fields for all the measures and dimension in the bound OLAP cube, and move these fields to the specified area to make them hidden. To do it, use the [PivotGridControl.RetrieveFields](https://docs.devexpress.com/WPF/DevExpress.Xpf.PivotGrid.PivotGridControl.RetrieveFields.overloads) method overload with the _visible_ parameter and set the field's visibility to `false`. The `RetrieveFields` method generates [DataSourceColumnBinding](https://docs.devexpress.com/WPF/DevExpress.Xpf.PivotGrid.DataSourceColumnBinding?v=22.1) objects for each Pivot Grid field in OLAP, Server, and Optimized [modes](https://docs.devexpress.com/CoreLibraries/403802/devexpress-pivot-grid-core-library/pivot-grid-modes?v=22.1).

```

File: Readme.md [hunk 3]
```diff
@@ -1,18 +1,76 @@
...
+   ```cs
+   public MainWindow() {
+         InitializeComponent();
+
+         // Retrieves fields.
+         pivotGridControl1.RetrieveFields(FieldArea.ColumnArea, false);
+   }
+   ```
+
+1. Place some of the created fields within corresponding Pivot Grid Control areas and set their [PivotGridField.Visible](https://docs.devexpress.com/WPF/DevExpress.Xpf.PivotGrid.PivotGridField.Visible) property to `true`.
```
Comment (dirk-pieterse) [inline Readme.md]: Some of the created fields? Which ones?

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md (hunk 3 above)
Comment (natakazakova) [reply Readme.md]:
```suggestion
1. Place the created fields within corresponding Pivot Grid Control areas and set their [PivotGridField.Visible](https://docs.devexpress.com/WPF/DevExpress.Xpf.PivotGrid.PivotGridField.Visible) property to `true`.

```

**Pull request merged**